pkg/sql/schemachanger/scpb/elements.proto

```diff
@@ -128,7 +130,15 @@ message ElementProto {
   CompositeTypeAttrType composite_type_attr_type = 140 [(gogoproto.moretags) = "parent:\"CompositeType\""];
   CompositeTypeAttrName composite_type_attr_name = 141 [(gogoproto.moretags) = "parent:\"CompositeType\""];
 
-  // Next element group start id: 160
+  // Function elements.
+  FunctionName function_name = 160 [(gogoproto.moretags) = "parent:\"Function\""];
+  FunctionVolatility function_volatility = 161 [(gogoproto.moretags) = "parent:\"Function\""];
+  FunctionLeakProof function_leak_proof = 162 [(gogoproto.moretags) = "parent:\"Function\""];
+  FunctionNullInputBehavior function_null_input_behavior = 163 [(gogoproto.moretags) = "parent:\"Function\""];
+  FunctionBody function_body = 164 [(gogoproto.moretags) = "parent:\"Function\""];
+  FunctionParamDefaultExpression function_param_default = 165 [(gogoproto.moretags) = "parent:\"Function\""];
+
```

Comment:
I saw `message Function` defined below. Why isn't here, as one of the element?

Reply:
it's at line 65 :), we put descriptor level elements at lower field ids.